packages/back-end/src/models/ApiKeyModel.ts

```diff
@@ -69,26 +85,108 @@ export function generateSigningKey(prefix: string = "", bytes = 32): string {
   );
 }
 
-export async function createApiKey({
+export async function createOrganizationApiKey({
+  organizationId,
+  description,
+  role = "readonly",
+}: {
+  organizationId: string;
+  description: string;
+  role: "admin" | "readonly";
+}): Promise<ApiKeyInterface> {
+  return await createApiKey({
+    organization: organizationId,
+    secret: true,
+    encryptSDK: false,
+    description,
+    environment: "",
+    project: "",
+    role,
+  });
+}
+
+export async function createUserPersonalAccessApiKey({
+  userId,
+  organizationId,
+  description,
+  environment,
+  project,
+}: {
+  userId: string;
+  organizationId: string;
+  description: string;
+  environment: string;
+  project: string;
```

Comment:
We don't need `environment` and `project` for these.  That's only for the legacy SDK keys